### DISCUSSION TITLE
[General] Fault Proof Thread!!!

### DISCUSSION THREAD

**Discussion by sbvegan** [ethereum-optimism/developers#446] [General]
### Discussion Topic

What are the things you want to know about fault proofs?

**Comment by sbvegan** [comment]
Adding @Chomtana's request: how can someone submit invalid output roots to OP Sepolia?

**Reply by cbfyi** [reply]
Also for completeness, how can someone produce, and submit a valid output root? 😄

**Comment by opfocus** [comment]
https://discord.com/channels/667044843901681675/1080862392281481246/1248870284035883098   a relevant context

**Comment by kush30** [comment]
i just bridged from OP to Base but my txn is stuck , Amount is deducted from OP but it not yet received 
![Uploading Screenshot_20240612_091702.jpg…]()

**Reply by Chomtana** [reply]
Official bridge doesn't permit you to bridge from Optimism to Base directly. If you use a 3rd party bridge, I advise you to contact them on the bridge's Discord.

This is not related to fault proof.

**Comment by elvinvalentino** [comment]
Hi guys i have a question. can we use other token other than ETH like custom ERC20 token for the challenger and attacker to play the game on FaultDisputeGame smart contract?
